db/queries/insert_queries.js

```diff
@@ -12,11 +12,17 @@ _location TEXT,
 _requester VARCHAR(25),
 _start_dt TIMESTAMP,
 _end_dt TIMESTAMP,
-_price MONEY
+_price NUMERIC(6, 2)
 */
+// TODO: Functions not working yet, made an insert
+// exports.ONE_TASK = `
+//     SELECT
+//         insert_one_task($1, $2, $3, $4, $5, $6, $7, $8)
+//     ;
+// `
 exports.ONE_TASK = `
-    SELECT
-        insert_one_task($1, $2, $3, $4, $5, $6, $7, $8)
+    INSERT INTO task (title, description, category_id, location, requester, start_dt, end_dt, price)
```

Comment:
@snguyenthanh This was buggy due to functions not working previously so I did this, change it back if you want, haha

Reply:
It was not the function's bug but duplicate functions by keeping on creating functions when the app starts.
The bug has been resolved by dropping all functions before creating any, when the app starts.
I will revert back to `SELECT insert_one_task($1, $2, $3, $4, $5, $6, $7, $8)`.